deploy/frontend.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: sed-frontend
```

Comment:
I may be well wrong, but I think this needs to match the appID in the cloud-servise-config